examples/hybrid/package.json

```diff
@@ -0,0 +1,59 @@
+{
```

Comment:
json files should be formatted with 2 spaces

Reply:
done